eng/pipelines/jobs/test-binaries.yml

```diff
@@ -32,6 +32,12 @@ jobs:
     disableComponentGovernance: true
     disableSbom: true
 
+    # Override containers for Linux to test on real clib and architecture variants
+    ${{ if eq(parameters.osGroup, 'Linux') }}:
+      container: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04
+    ${{ if eq(parameters.osGroup, 'Linux_Musl') }}:
+      container: mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-WithNode
```

Comment:
I know this is the image we used before, but why do we need node?

Reply:
Azure Pipelines requires containers to have node on them in order to execute steps within the image.